Task7/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "weathr-app",
+  "version": "1.0.0",
+  "description": "A webpack starter for Angular",
+  "scripts": {
+    "start": "webpack-dev-server --hot --progress",
+    "dev-serve": "http-server ./ -a 0.0.0.0 -p 9001",
+    "prod-serve": "http-server ./ -a 0.0.0.0 -p 9000",
+    "ngc": "ngc -p src",
```

Comment:
Please configure it properly.  Do not put generated files into working folder. 

https://angular.io/docs/ts/latest/cookbook/aot-compiler.html#!#compile